magma_access_gateway_installer/__init__.py

```diff
@@ -142,18 +152,20 @@ def validate_arbitrary_dns(args: argparse.Namespace):
 
 
 def validate_static_ip_allocation(args: argparse.Namespace):
```

Comment:
Here the validation is expecting the complete args as the other validations are expecting arg values (ex. s1_ip_address). I think we should use only what's needed so this method would require sgi_ipv6_address, sgi_ipv6_gateway, ...)